ui/component/fileDrop/view.jsx

```diff
@@ -80,9 +80,6 @@ function FileDrop(props: Props) {
           navigateToPublish();
           updatePublishForm({
             filePath: selectedFile.path || selectedFile.name,
-            fileDur: 0,
-            fileSize: 0,
-            fileVid: false,
```

Comment:
The purpose of these lines was to do the math to guess Mbps and judge whether file needs to be transcoded (is a video, and size is large compared to duration). Is that done somewhere else now?

Reply:
@jessopb yes, it was removed so this logic happens only in one place (publishFile/view.jsx)

Reply:
Ok. The messaging seems wrong. `Your video may not be the best format.` is always the message which is the message for (isVid && !duration) so something isn't working.